package/upgrade/extra_manifests/whereabouts.cni.cncf.io_ippools.yaml

```diff
@@ -0,0 +1,71 @@
+
```

Comment:
@bk201 wondering if you know whether removing this file in the future version will also delete the related CRDs?

Reply:
Guess not. The script applies everything it found under `extra_manifests`.

Reply:
great thanks.